ajax_select/lookup_channel.py

```diff
@@ -97,7 +97,7 @@ def get_objects(self, ids):
         pk_type = self.model._meta.pk.to_python
         ids = [pk_type(pk) for pk in ids]
         things = self.model.objects.in_bulk(ids)
-        return [things[aid] for aid in ids if aid in things]
+        return things.values()
```

Comment:
I'm referring to line 100: `[things[aid] for aid in ids of aid in things]`

That should make sure that the UUID are returned in the same order that the ids were requested in. This is important if there is any enforced sort order.  Because `in_bulk` does not preserve that order.